packages/core-sdk/src/types/config.ts

```diff
@@ -12,7 +12,9 @@ export type SupportedChainIds =
   | "mainnet"
   | "80001"
   | "mumbai"
-  | "polygonMumbai";
+  | "polygonMumbai"
```

Comment:
We can delete mumbai and polygonMumbai  and mainnet related config. We are not supporting them in our next sdk release